src/NoFrills.js

```diff
@@ -8,17 +8,19 @@ class NoFrills {
 
```

Comment:
Does this file have an associated test file? Are the new methods you are adding getting tested?